tomcat/src/main/java/org/apache/coyote/http11/request/requestline/HttpMethod.java

```diff
@@ -12,9 +15,12 @@ public static HttpMethod from(String data) {
         if ("POST".equals(data.toUpperCase().strip())) {
```

Comment:
```suggestion
        Arrays.stream(values()).filter(httpMethod -> httpMethod.method == (method.toUpperCase().strip()))
                .findFirst()
                .orElseThrow(()->new IllegalArgumentException(""));
```

이와 비슷하게도 가능할 꺼 같아요. 지금 로직상 메소드 하나 늘어날 때마다 작업이 늘거 같습니당

Reply:
좋네요~!